### ISSUE TITLE
Unstable

### ISSUE THREAD

**Issue by Mir-2002** [bacontsu/multibox#2]
I tried it and it was pretty unstable. I couldnt switch between windows properly and had to always manually kill the task through task manager.

**Comment by bacontsu** [comment]
it is pretty unstable indeed, this was the outdated version of multiboxing and the current version is pretty much broken because I'm trying to fix it.
this thing, the hook were global and I cannot wrap my head around it, aswell i as I start abandoning the multibox and start using full autofarmer anyways. you can read my issue on the base source code here :
https://github.com/Blacklock/gtautofarmer/issues/9

sadly, the only way to switch between windows is using alt + tab. 
you don't have to endtask the program or using alt + f4 to close it, just select the (Multibox) window with alt+tab and then Click "Turn Off Multibox" or just press Ctrl + K. this is only the fix for rn, until I finally find the actual fix. on the base program you cant even use your pc anymore after turning on Multibox, it hooks up your keyboard and pc only for growtopia and the only way is to force-shutdown your pc. 

Thanks for the report, althou I'm pretty aware of this bug and alr trying to fix this since november.

**Comment by Mir-2002** [comment]
Yeah i noticed that my kbm only responded to growtopia too. Anyways, goodluck on stabilizing it. I'll keep an eye on this one since i badly need it too.

**Comment by bacontsu** [comment]
> Yeah i noticed that my kbm only responded to growtopia too. Anyways, goodluck on stabilizing it. I'll keep an eye on this one since i badly need it too.

aight thank you very much for following this project, imma try to fix it
